tests/tests_units/rules/test_models/test_information_rules.py

```diff
@@ -46,6 +46,53 @@ def invalid_domain_rules_cases():
         "Value error, Metadata.role should be equal to 'information architect'",
         id="invalid_role",
     )
+    yield pytest.param(
+        {
+            "Metadata": {
+                "role": "information architect",
+                "creator": "Jon, Emma",
+                "contributor": "David",
```

Comment:
Is there a reason to distinguish between creator and contributor?

Reply:
Good point, we can simply call them all creator!